apps/microtvm-vm/base-box/packer.json.template

```diff
@@ -0,0 +1,43 @@
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+{
+    "variables": {
+        "provider": null,
+        "version": null,
+        "access_token": null,
+        "skip_add": "false"
+    },
+    "builders": [
+        {
+            "type": "vagrant",
+            "output_dir": "output-vagrant-{{user `provider`}}",
+            "communicator": "ssh",
+            "source_path": "generic/ubuntu2004",
+            "provider": "{{user `provider`}}",
+            "template": "Vagrantfile.packer-template",
+            "skip_add": "{{user `skip_add`}}"
+        }
+    ],
+    "post-processors": [
+        {
+            "type": "vagrant-cloud",
+            "box_tag": "areusch/microtvm-staging",
```

Comment:
What is areusch/microtvm-staging ? Should we be merging this in ? 

If we are putting this into main - I don't think this should have any information of this sort here.